index.d.ts

```diff
@@ -100,6 +100,9 @@ declare namespace ExpressValidator {
     isBase64?: ValidatorSchemaOptions
     isDataURI?: ValidatorSchemaOptions
     isWhitelisted?: ValidatorSchemaOptions
+    isMimeType?: ValidatorSchemaOptions
+    isPort?: ValidatorSchemaOptions
+    isISO31661Alpha2?: ValidatorSchemaOptions
```

Comment:
So, there's more to the TS typings: this is only adding support in the validation schemas.

We need to change the `Validator` interface, so that normal validators in the check and legacy APIs work, and also add tests for these cases (look at the `.ts` files around to find about them).